root/release/edit/tracklist.tt

```diff
@@ -349,8 +349,16 @@
     <!-- /ko -->
 
     <div data-bind="visible: loaded() && !collapsed()">
+
+      <p>
+        <label>
+          <input id="tracks-unknown" type="checkbox" data-bind="checked: tracksUnknownToUser, enable: tracks().length === 0" />
```

Comment:
Would it make sense to hide the checkbox if there are tracks? 'Cause you can't check it unless you remove them all anyway. (If you were able to remove all the existing tracks by checking it, hiding the tracklist editor entirely, that'd make a bit more sense to show it to me, but.)

Reply:
That's actually a very good point. Made the change.